src/ifp/test/helpers.py

```diff
@@ -0,0 +1,8 @@
+import odb
+
+def make_rect(design, xl, yl, xh, yh):
+    xl = design.micronToDBU(xl)
```

Comment:
It is much less error prone to work in DBU than in floating point microns.  Look at the number of places in OR that are snapping user inputs.  I think it better if user applications do the same rather than introduce a source of bugs.  We can provide helpers for those who really like microns.  What do you think?

Reply:
oh I see, so the expectation would be that user work in DBUs by default/most of the time.

Looking at the db initializer, it looks like DBU originally comes from the Tech files:
https://github.com/The-OpenROAD-Project/OpenROAD/blob/816cfe95a0c90dc7fe0395c000327b3aa6babf96/src/odb/src/db/dbLib.cpp#L366

So maybe it would make sense to expose the conversion there instead?
```
tech = Tech()
tech.make_rect_um()
```

Reply:
Isn't `Tech` required to do the micron dbu conversion? (because you need `_dbu_per_micron`).